ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -412,7 +422,9 @@ internal class CustomerCenterViewModelImpl(
                     currentState
                 }
             }
+            return true
```

Comment:
So here we say we've loaded the promo offer, but it could be that we haven't in case the state is not `Success` right? It might be ok in that case though so 🤷

Reply:
hmm you're right, I think I can return only when it's actually loaded